products/vmware-harbor-registry.md

```diff
@@ -8,8 +8,11 @@ changelogTemplate: https://docs.vmware.com/en/VMware-Harbor-Registry/services/vm
 releaseDateColumn: true
 eolColumn: End of general support
 
-releases:
+identifiers:
+-   cpe: cpe:2.3:a:linuxfoundation:harbor
+-   cpe: cpe:/a:linuxfoundation:harbor
 
```

Comment:
```suggestion
```

Not sure about this one, the page explicitly tracks the VMWare Harbor Registry, not the underlying OSS project. WDYT ?